proto/server.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package server;
+
+option go_package = "./;server";
+
+message PublishStreamRequest {
+  string channel = 1;
+  bytes data = 2;
+}
+
+message PublishStreamResponse {}
+
+service Grafana {
+  rpc PublishStream(PublishStreamRequest) returns (PublishStreamResponse);
```

Comment:
```suggestion
  rpc PublishStream(stream PublishStreamRequest) returns (PublishStreamResponse);
```
channel should be empty in message 2+

Reply:
My idea here was introducing a possibility to publish arbitrary data to arbitrary channels (with permission check on server side on every call). If we want stream then maybe we should add additional method here. Call it `RunStream` to mirror plugin API, send StreamPacket, and pass `channel` in initial metadata to avoid confusing requirement with empty channel after first request.

Reply:
that makes sense